Example/Tests/GINISupportedFormatsViewControllerTests.swift

```diff
@@ -29,7 +33,7 @@ final class GINISupportedFormatsViewControllerTests: XCTestCase {
     
     override func setUp() {
         super.setUp()
-
+        GiniConfiguration.sharedConfiguration = initialGiniConfiguration
```

Comment:
Isn't this redundant? The configuration will be set at the beginning of the test either way. And before that, either way it will be equal to the initial one.